### PR TITLE
Login signup

### PR DESCRIPTION
**Pull request by srishtigautam21**
Added footer 
Made the login and signup page

### CODE REVIEW

**Review by DivyanshSareen — APPROVED** [review]
Everything looks great. Good to merge. I have a suggestion for the UI, the signup and login options can stand out with primary color. 
image

File: Pages/LoginPage/login.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.login-signup-page{
+    display:flex;
+    justify-content: center;
+    align-items: center;
+    margin:3rem 0 3rem 0;
```
Comment (iRohitGaur) [inline Pages/LoginPage/login.css:5]: this can be done with "margin: 3rem 0;"

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: Pages/LoginPage/login.css (hunk 1 above)
Comment (srishtigautam21) [reply Pages/LoginPage/login.css:5]: ok will change this

**Pull request merged**